evap/static/css/evap.scss

```diff
@@ -1004,6 +1004,77 @@ a.no-underline:hover {
 }
 
 
+/* custom checkboxes (based on http://getbootstrap.com/docs/4.0/components/forms/#custom-forms) */
+.form-check-input {
+    position: absolute;
+    z-index: 1;
+    width: 1rem;
+    height: 1rem;
+    opacity: 0;
+    &:checked ~ .form-check-label::after {
+        opacity: 1;
+    }
+    &:checked ~ .form-check-label::before {
+        background-color: $evap-dark-blue;
+    }
+    &:hover ~ .form-check-label::before {
+        background-color: darken($light-gray, 10%);
+    }
+    &:checked:hover ~ .form-check-label::before {
+        background-color: darken($evap-dark-blue, 10%);
+    }
+    &:focus ~ .form-check-label::before {
+        box-shadow: 0 0 0 0.2rem rgba($evap-dark-blue, 0.25);
+    }
+    &:disabled {
+        ~ .form-check-label {
+            color: $light-gray;
+            &::before {
+                background-color: $lighter-gray;
+                box-shadow: none;
+            }
+        }
+    }
+}
+input[type="hidden"] ~ .form-check-label {
+    display: none;
+}
```

Comment:
is this the part that fixes the delete buttons? maybe add some comment about why this is necessary?

Reply:
check